parley/tests/current/.gitignore

```
@@ -0,0 +1 @@
+*.png
```

```suggestion
*.png

```
Our style is to always have a final newline.